helm_deploy/laa-estimate-eligibility/values/ccq-staging.yaml

```diff
@@ -0,0 +1,90 @@
+replicaCount: 5
+
+resources: {}
+
+image:
+  repository: nginx
+  tag: stable
+  pullPolicy: IfNotPresent
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: false
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: "cd-serviceaccount"
+
+podSecurityContext: {}
+# fsGroup: 2000
+
+securityContext:
+ runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  hosts:
+    - host: main-ccq-staging.cloud-platform.service.justice.gov.uk
+      secret: false
+#    - host: check-your-client-qualifies-for-legal-aid-staging.cloud-platform.service.justice.gov.uk
+#      secret: false
+#    - host: staging.check-your-client-qualifies-for-legal-aid.service.gov.uk
+#      secret: ccq-staging-cert-secret
+  annotations:
+    external-dns.alpha.kubernetes.io/set-identifier: "check-client-qualifies-laa-estimate-eligibility-check-client-qualifies-staging-green"
+    external-dns.alpha.kubernetes.io/aws-weight: "0"
+
+postgresql:
+  enabled: false
+
+notifications:
+  errorMessageTemplateId: 928f3ea3-0134-4966-b43d-ba7e1369fd5d
+  recipient: patrick.gleeson@digital.justice.gov.uk,michael.blatherwick@digital.justice.gov.uk,william.clarke@digital.justice.gov.uk
+
+googleTagManager:
+  containerId: GTM-WJ98FR4
+
+cfe:
+  host: http://cfe-civil.cfe-civil-staging.svc.cluster.local
+  environment_name: staging
+
+featureFlags:
+  sentry: ENABLED
+  overrides: NOT_ENABLED
+  indexProduction: NOT_ENABLED
+  maintenanceMode: NOT_ENABLED
+  basicAuthentication: ENABLED
+  welshCw: NOT_ENABLED
+
+geckoboard:
+  metricsDataset: "ccq_monthly_metrics_staging"
+  allMetricsDataset: "ccq_all_metrics_staging"
+  lastPagesDataset: "ccq_last_pages_staging"
+  validationsDataset: "ccq_validations_staging"
+  allJourneysDataset: "ccq_all_journeys_staging"
+  monthlyJourneysDataset: "ccq_monthly_journeys_staging"
+  recentJourneysDataset: "ccq_recent_journeys_staging"
+  enabled: not_enabled
+
+sentry:
+  # Despite looking very secret-y, the key here is _not_ a secret, as it will appear in every HTTP response header
+  cspReportEndpoint: https://o345774.ingest.sentry.io/api/6763263/security/?sentry_key=fda2f0cddce3417c87e4df62f3611e76
+  # Similarly this is available in the page source
+  dsn: https://f55bec8a5587418e932cd4935569a06b@o345774.ingest.sentry.io/6763263
+
+app:
+  primaryHost: ""
+  seedAdmins: patrick.gleeson@digital.justice.gov.uk,michael.blatherwick@digital.justice.gov.uk,william.clarke@digital.justice.gov.uk,masum.khan@digital.justice.gov.uk,hettie.street@digital.justice.gov.uk
+
+google:
+  oauthClientId: 276075615370-ncqvmdi84jruv1jggfg4bu8nlvk1drj0.apps.googleusercontent.com
+  oauthRedirectUri: https://staging.check-your-client-qualifies-for-legal-aid.service.gov.uk/auth/subdomain_redirect
```

Comment:
I feel like this should strictly be a host that's controlled by this namespace, but it would only be a problem if we deleted the old namespace before we transferred control of this host to this namespace, which we won't do, so let's not worry about it for now.